fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectoryWrapper.java

```diff
@@ -111,7 +111,7 @@ public DirectoryReader getWriterReader(boolean flush) throws IOException {
         return writerReader;
     }
 
-    private static class FDBDirectoryMergeScheduler extends ConcurrentMergeScheduler {
+    private static class FDBDirectoryMergeScheduler extends SerialMergeScheduler {
```

Comment:
`SerialMergeScheduler` has a `synchronized` block around the `merge` method, to prevent multiple threads from doing a merge at the same time.
Do we really want that? Would it make more sense for multiple requests to do a `merge` to be able to execute the merge at the same time?
`ConcurrentMergeScheduler` also has a `synchronized` on its `merge` method, so we may need to synchronize around getting the next merge.

Reply:
✅